src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1810,14 +1810,66 @@ void Graph::resolveInPlaceDirection(const NodePtr& node) const {
                     return InplaceDirectionType::NONE;
                 };
                 auto result = searchNonCyclicDirection(node, inPlaceInpPort);
-                if (one_of(result, InplaceDirectionType::UP, InplaceDirectionType::NONE)) {
+                if (InplaceDirectionType::UP == result) {
                     auto config = node->getSelectedPrimitiveDescriptor()->getConfig();
                     config.inConfs[inpPort].inPlace(-1);
                     node->initDescriptor(config);
                 } else if (InplaceDirectionType::DOWN == result) {
                     auto config = node->getSelectedPrimitiveDescriptor()->getConfig();
                     config.outConfs[inPlaceInpPort].inPlace(-1);
                     node->initDescriptor(config);
+                } else if (InplaceDirectionType::NONE == result) {
+                    // resolve cyclic inplace to downstream instead of upstream for the node
+                    // when there is only one output referencing to the edges of it,
+                    // thus benefits zero-copy of outputs.
+                    int numConflicts = 0;
```

Comment:
It looks like `numConflicts` is never decremented. Better to use unsigned type, e.g. `size_t`.

Reply:
done